sentry/const.py

```diff
@@ -36,7 +36,9 @@ def to_float_if_defined(value):
     getattr(odoo.loglevels, "LOG_%s" % x): getattr(logging, x)
     for x in ("CRITICAL", "ERROR", "WARNING", "INFO", "DEBUG", "NOTSET")
 }
-DEFAULT_LOG_LEVEL = "warn"
+DEFAULT_BREADCRUM_LOG_LEVEL = "info"
+
+DEFAULT_EVENT_LOG_LEVEL = "error"
```

Comment:
I'm wondering the impact on existing instance using previous default values :thinking: